.github/workflows/manual-versioning.yml

```diff
@@ -16,26 +16,31 @@ jobs:
   release:
     runs-on: ubuntu-latest
     concurrency: release
-    permissions:
-      id-token: write
-      contents: write
 
     steps:
+      - name: "Generate token"
+        id: generate_token
+        uses: tibdex/github-app-token@v1
```

Comment:
Should we use this instead: https://github.com/actions/create-github-app-token

This one - `tibdex/github-app-token@v1` - looks to have recently been marked as archived.